x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/pivot/Pivot.java

```diff
@@ -72,7 +73,7 @@ public Pivot(PivotConfig config) {
     public void validateConfig() {
         for (AggregationBuilder agg : config.getAggregationConfig().getAggregatorFactories()) {
             if (Aggregations.isSupportedByDataframe(agg.getType()) == false) {
-                throw new RuntimeException("Unsupported aggregation type [" + agg.getType() + "]");
```

Comment:
I changed all of these from `RuntimeException` to `ElasticsearchException` as they probably should never have been an `RuntimeException` due to serialization between nodes and making sure we only catch and unhandled exceptions we throw on purpose.